families/supplychain/restapi/openapi.yaml

```diff
@@ -0,0 +1,398 @@
+swagger: '2.0'
+
+info:
+  version: "0.8.0"
+  title: Sawtooth Supplychain REST API
+  description: |
+    _This HTTP pragmatic REST API is built on top of Sawtooth Lake's existing
+    ZMQ/Protobuf infrastructure, simplifying SupplyChain client interactions
```

Comment:
If this is getting built against the ZMQ/Protobuf interface, it will require significant rewriting of existing functionality. The correct answer is probably to refactor the REST API so it's key components are exportable in some sort of REST API SDK, or to have this API submit HTTP requests to a REST API instead of using ZMQ/Protobuf directly.

I don't hate the idea of a REST API SDK, but that is a sprint or two worth of work, so I feel like building this on top of the existing REST API is the correct answer here.